tests/functional/sync_v3_sign_up.js

```diff
@@ -253,6 +255,44 @@ define([
         .then(testElementExists(selectors.CHOOSE_WHAT_TO_SYNC.HEADER))
         .then(testElementExists(selectors.CHOOSE_WHAT_TO_SYNC.ENGINE_ADDRESSES))
         .then(testElementExists(selectors.CHOOSE_WHAT_TO_SYNC.ENGINE_CREDIT_CARDS));
-    }
+    },
+
+    'verify from original tab\'s P.O.V., Fx <= 56': function () {
+      return this.remote
+        .then(openPage(SIGNUP_FX_56_PAGE_URL, selectors.SIGNUP.HEADER, {
+          webChannelResponses: {
+            'fxaccounts:can_link_account': { ok: true },
+            'fxaccounts:fxa_status': { capabilities: null, signedInUser: null }
+          }
+        }))
+        .then(fillOutSignUp(email, PASSWORD))
+        .then(testElementExists(selectors.CHOOSE_WHAT_TO_SYNC.HEADER))
+        .then(click(selectors.CHOOSE_WHAT_TO_SYNC.SUBMIT))
+
+        .then(testElementExists(selectors.CONFIRM_SIGNUP.HEADER))
+
+        .then(openVerificationLinkInDifferentBrowser(email))
+
+        .then(noPageTransition(selectors.CONFIRM_SIGNUP.HEADER, 5000));
```

Comment:
nit: The comment about `about:accounts` taking over might be useful for our future selves here.